build64/Makefile

```diff
@@ -66,6 +213,8 @@ APP_OBJS = app/badram.o \
 
 OBJS = boot/startup.o boot/efisetup.o $(SYS_OBJS) $(LIB_OBJS) $(TST_OBJS) $(APP_OBJS)
 
+debug: clean memtest.debug memtest.efi
+
```

Comment:
The `debug` target should come after the `all` target, to keep `all` as the default.

Reply:
You are right. I have changed it